webapp/components/product-info.html

```diff
@@ -25,15 +25,21 @@
       m.set('msedge', 'MS Edge');
       return m;
     })();
-    /*eslint-disable no-unused-vars*/
-    const VERSION_PATTERNS = {
-      major: /(\d+)/,
-      majorAndMinor: /(\d+\.\d+)/
-    };
-    const DEFAULT_BROWSER_NAMES = ['chrome', 'edge', 'firefox', 'safari'];
-    const DEFAULT_PRODUCT_SPECS = Array.from(DEFAULT_BROWSER_NAMES);
-    const DEFAULT_PRODUCTS = DEFAULT_PRODUCT_SPECS.map(p => parseProductSpec(p));
-
+    Object.defineProperty(window.wpt, 'VersionPatterns', {
+      value: Object.freeze({
+        Major: /(\d+)/,
+        MajorAndMinor: /(\d+\.\d+)/,
+      }),
+    });
+    Object.defineProperty(window.wpt, 'DefaultBrowserNames', {
+      get: () => ['chrome', 'edge', 'firefox', 'safari'],
```

Comment:
nit: Why create a new Array every time? `Object.freeze()` should work on an array.